models/biocro/Dockerfile

```diff
@@ -1,3 +1,6 @@
+# this needs to be at the top, what version are we building
+ARG IMAGE_VERSION="latest"
```

Comment:
Should image version be `develop`? I am getting the error : `manifest for pecan/models:latest not found`.

Reply:
You can use `docker build --build-arg IMAGE_VERSION=develop -t pecan/model-biocro .`

Reply:
Ok.